pywbem/cim_operations.py

```diff
@@ -587,6 +590,25 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
             Initial enablement status for maintaining statistics about the
             WBEM operations executed via this connection. See the
             :ref:`Statistics` section for details.
+
+          enable_logging (:term:`string` or None):
+            *New in pywbem 0.12 as experimental
+            Enable logging of operation requests and/or xml requests and
+            responses and set the detail level for the logging.
+
+            * If None logging is not enabled.
+
+            * If 'min' the logging is set to capture to a maximum log size
+              defined by the `DEFAULT_MAX_LOG_ENTRY_SIZE` variable.
+
+            * If 'all' the full data content is captured.
+
+            Which of the log points is captured is determined by the
+            characteristics set for logger names as defined in the documentation
+            section on logging.
```

Comment:
* Link to the logging section.
* "log points" is not a defined term. I suggest using the same terms used above, i.e. "API calls and CIM-XML requests and responses".

Reply:
Cleaned up logging points.  However, not sure how to create link to doc section of anther module.
DISCUSSION

Reply:
How to link to a section (in same or other module):
```
:ref:`section title`
```